pkg/network/network.go

```diff
@@ -206,38 +211,44 @@ func defaultConfig() *Config {
 
 // NewConfigFromConfigMap creates a Config from the supplied ConfigMap
 func NewConfigFromConfigMap(configMap *corev1.ConfigMap) (*Config, error) {
+	return NewConfigFromMap(configMap.Data)
+}
+
+// NewConfigFromMap creates a Config from the supplied data map.
+func NewConfigFromMap(data map[string]string) (*Config, error) {
 	nc := defaultConfig()
-	if _, ok := configMap.Data[IstioOutboundIPRangesKey]; ok {
+	if _, ok := data[IstioOutboundIPRangesKey]; ok {
 		// TODO(0.15): Until the next version is released, the validation check is
 		// enabled to notify users who configure this value.
-		logger := logging.FromContext(context.Background()).Named(configMap.Name)
+		logger := logging.FromContext(context.Background()).Named("config-network")
```

Comment:
Why inline?  Can we still use a constant?

Reply:
The object does not exist anymore :)

Reply:
But this is a low-key logging as well, so I don't think we really care, given it'll be removed soon.